content/next/index.md

```diff
@@ -1,8 +1,9 @@
 ---
 draft: true
+aliases: ["/en/"]
```

Comment:
This one should have the alias 😄

Reply:
I'm not quite sure what this value should be? I was copying the old document in this folder.